include/SIMD.h

```diff
@@ -28,6 +28,9 @@ FORCE_INLINE int popcnt32(int x32)
 
 #if defined(USE_AVX) || defined(USE_AVX2)
 #include <immintrin.h>
+#elif defined __aarch64__
+#warning "--- THIS IS AARCH64"
+#include <sse2neon.h>
```

Comment:
We are going to need to support Neon one way or the other. I am pondering if this is the right approach though, or if we should implement the Neon intrinsics directly? If it saves us time in the short run, maybe a viable option?